### PR TITLE
Add github image link in the readme

### LINKED ISSUE #239: Issues with releases and assets
1. PyPI does not allow links to local assets in its documentation, so the logo shows up as a missing image.
2. ~~The inclusion of assets in our release file has doubled its size (from 25K to 50K).~~ (edit: actually, no)

From other repositories I have inspected, the README typically has a link to the content on github rather than a local path.

@moy How do you feel about excluding `assets` from the release and linking to our assets on github instead? This probably deserves a post1 release.

### PR DESCRIPTION
**Pull request by QuentinSanchez**
Fixes #239

    Add code in order to concatenate the github logo raw link, to the
    relative one presents in the readme.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,16 +12,20 @@ def requirements_from_file(filename):
...
 
-def read(fname, URL):
+def read(fname, URL, URLImage):
     """Read the content of a file."""
     readme = open(path.join(path.dirname(__file__), fname)).read()
     if hasattr(readme, 'decode'):
         # In Python 3, turn bytes into str.
         readme = readme.decode('utf8')
     # turn relative links into absolute ones
+
```
Comment (moy) [inline setup.py]: This new newline is useless and counter productive: it takes the comment above apart from the corresponding lines of code.

File: setup.py [hunk 2]
```diff
@@ -12,16 +12,20 @@ def requirements_from_file(filename):
...
     readme = re.sub(r'`<([^>]*)>`__',
                     r'`\1 <' + URL + r"/blob/master/\1>`__",
                     readme)
+
+    readme = re.sub(r"\.\. image:: /", ".. image:: " + URLImage, readme)
```
Comment (moy) [inline setup.py]: I'd re-add the /, in case someone ever uses a URL without a trailing slash.

File: setup.py [hunk 3]
```diff
@@ -22,6 +22,9 @@ def read(fname, URL):
     readme = re.sub(r'`<([^>]*)>`__',
                     r'`\1 <' + URL + r"/blob/master/\1>`__",
                     readme)
+
+    readme = re.sub(r"\.\. image:: /", ".. image:: " + URLImage+"/", readme)
```
Comment (moy) [inline setup.py]: Spaces around `+` please.

**Comment by moy** [comment]
BTW, please post a message explicitly when you force-push to a PR, we don't receive notification and don't know how you've dealt with remarks otherwise.

**Comment by QuentinSanchez** [comment]
i've made a forced push, fixed spaces around + and change the commit's author name .

**Pull request merged**